### DISCUSSION TITLE
How to use the App Router feature in Nextjs13.4 instead of the Pages Router?

### DISCUSSION THREAD

**Discussion by tuanfront-end** [wpengine/faustjs#1504] [Q&A] [answered]
Hi, thank you so much for this amazing project.
May I hear your thoughts on this matter? Is it possible to use App Router instead of the Pages Router now?
I tried but there are errors, I don't know if I am wrong somewhere or currently, Faustjs is not suitable to use App Router instead of the Pages Router?
If possible, that would be great, because App Router is the future, and can always use the latest features of Nextjs.
Thank you so much!

**Comment by theodesp** [comment, marked as answer]
Hey @tuanfront-end thanks for asking. We are looking to provide support for  App Router this quarter since Next.js is transitioning towards this approach. We have posted some updates in our blog:

https://faustjs.org/blog

I will move this to a conversation thread.
